.eslintrc.cjs

```diff
@@ -139,6 +139,7 @@ module.exports = {
                 'ts-check': false,
             }
         ],
+        '@typescript-eslint/prefer-namespace-keyword': 'error',
```

Comment:
this change now needs to be made on the microsoft/definitelytyped-tools repo in the eslint-plugin package.